src/main/java/com/qaprosoft/carina/demo/gui/components/LoginForm.java

```diff
@@ -25,16 +26,17 @@
     @FindBy(xpath = ".//i[@class='head-icon icon-signout']")
     private ExtendedWebElement logOutIconButton;
 
-    public Login(WebDriver driver, SearchContext searchContext) {
-        super(driver, searchContext);
+    public LoginForm(WebDriver driver) {
+        super(driver);
     }
 
-    public boolean isLoginFormPresent() {
-        return loginForm.isElementPresent();
+    public LoginForm(WebDriver driver, SearchContext searchContext) {
+        super(driver, searchContext);
     }
 
-    public void openFieldsForLogIn() {
+    public void openLoginForm() {
```

Comment:
Are you sure what this method should return void?

Reply:
Yes, when I click on "Log In" icon, opening form for login and page don't change.